### DISCUSSION TITLE
Opening New Tabs in Same Working Directory on Mac

### DISCUSSION THREAD

**Discussion by eric-hemasystems** [nushell/nushell#11021] [Q&A] [answered]
Nushell has [excellent documentation](https://www.nushell.sh/book/default_shell.html) on how to make Nushell the default shell for many platforms. My only hang-up is on the Mac platform you can specify that new tabs should open in the same working directory (in fact this is the default):

!["New tabs open with" option](https://github.com/nushell/nushell/assets/34583914/4f8fb48d-1e9a-4107-9c61-d09e6718e92f)

This works just fine in things like zsh and bash. But for Nushell this doesn't work. This means anytime I want to start a new tab that tab ends up back at my `$HOME` instead of my `$PWD`. Any idea how I make that option work? I assume some option is being passed to the shell to specify the directory by the mac terminal program although I haven't found documentation on what that is.

**Comment by fdncred** [comment]
Is this for MacOS's terminal.app or some other terminal?

**Reply by eric-hemasystems** [reply]
Yes, it is for the default terminal app in MacOS.

**Comment by fdncred** [comment, marked as answer]
Maybe you don't have shell_integration turned on? It's a setting in the config.nu file.

**Reply by eric-hemasystems** [reply]
I gave this a try and it seems to have worked. It didn't right away but when I completely closed terminal then tried again it did. I wonder if this would be something good to mention on the default shell documentation page for other newbies like myself?

**Reply by fdncred** [reply]
Added clarification in documentation is good. It used to be enabled by default but someone didn't like it so they turned it off.
